less/bin-basket.less

```diff
@@ -0,0 +1,187 @@
+@bin-basket-circle-size: 150px;
+bin-basket {
+  cursor: pointer;
+  &.modern-basket {
+    >a {
+      background: @brand-primary;
+      color: white;
+      display: flex;
+      position: fixed;
+      right: 0;
+      top: 0;
+      -webkit-border-bottom-left-radius: 10px;
+      -moz-border-radius-bottomleft: 10px;
+      border-bottom-left-radius: 10px;
+      -webkit-box-shadow: 0px 0px 24px -3px rgba(0, 0, 0, 0.7);
+      -moz-box-shadow: 0px 0px 24px -3px rgba(0, 0, 0, 0.7);
+      box-shadow: 0px 0px 24px -3px rgba(0, 0, 0, 0.7);
+      padding: 8px 10px;
+      padding-right: 30px;
+      i {
+        font-size: inherit;
+      }
+      span {
+        font-size: 15px;
+      }
+    }
+    &.isOverTreshold {
```

Comment:
Stick to kebap case for css classes like all the other styling

Reply:
I will change the casing, the reason this is still here because in the future I will be implementing these to be used, but for now it isn't used anywhere.